app/ptokens/templates/buy_a_token.html

```diff
@@ -0,0 +1,182 @@
+                  </p>
+                  <p class="my-3">
+                    {%trans "Enjoy a vetted redemption cycle directly in Gitcoin - providing you with an end-to-end experience for working with our top talent." %} <br>
+                  </p>
+
+                  <p class="my-3">
+                    {%trans "Ready to dive in? Check out our top creators!" %} <br>
+                  </p>
+
+                  <a class="btn btn-gc-blue" href="{% url "users_directory" %}">{%trans "Explore" %}</a>
+
+                  <p class="my-4">
+                      {% trans "Disclaimer: Gitcoin personal tokens are currently in beta. Creators remain full control over whether or not they accept a redemption request." %}
+                      {% trans "There is an inherent risk that your redemption may not be redeemed, so please purchase at your own risk with the knowledge that secondary liquidity may be limited."%}
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div id="section-start" class="tab-section">
+            <div class="container px-0">
+              <div class="row mb-3 py-2 px-md-0 mx-sm-0">
+                <div class="col-12 col-md-12 px-0">
+                  <h1 class="font-title">{%trans "Make a Token!" %}</h1>
+                  <p class="my-3">
+                    {%trans "Looking to bootstrap your future potential?" %} <br>
+                  </p>
+
+                  <h1 class="font-title">{%trans "Get Paid Today for Work Tomorrow" %}</h1>
+                  <p class="my-3">
+                    {%trans "Receive funding up-front from those eager to work with you." %} <br>
+                  </p>
+                  <p class="my-3">
+                    {%trans "Customize your skillset to match where you excel." %} <br>
+                  </p>
+
+                  <h1 class="font-title">{%trans "Control Your Worth" %}</h1>
+                  <p class="my-3">
+                    {%trans "Determine your own rates and keep 100% of the profits." %} <br>
+                  </p>
+                  <p class="my-3">
+                    {%trans "Change your costs to respond to trends and client demand." %} <br>
+                  </p>
+
+                  <h1 class="font-title">{%trans "Solidify Your Reputation" %}</h1>
+                  <p class="my-3">
+                    {%trans "Resumes and CV’s are a thing of the past." %} <br>
+                  </p>
+                  <p class="my-3">
+                    {%trans "Let Gitcoin’s coveted reputation act as a glowing example of your standing in the ever-evolving digital world." %} <br>
+                  </p>
+
+                  <p class="my-4">
+                    {%trans "What are you waiting for?" %} <br>
+                  </p>
+
+                  <a class="btn btn-gc-blue" href="{% url "explorer" %}">{%trans "Get Started" %}</a>
```

Comment:
Looks like the dashboard is fixed in commit f62b4e21842edd6d633340437e7ac3bb5a44f133. We can cherry pick that commit and change this to `{% url "dashboard" %}`, then this'll be good to go!

Reply:
Hey, I can't see the dashboard on that commit. Care to clarify?

Reply:
I had to do a hard refresh to clear the browser cache for it to take effect

Reply:
I don't understand what you mean by that. What did you do hard reset on?